tools/transformer/src/wrappers/apifile.js

```diff
@@ -3,6 +3,7 @@ const { readFileSync, writeFileSync } = require("fs");
 
 /**
  * Read and parse API file as json
+ *
```

Comment:
[q] I'm curious to know what the thought process behind naming this "wrappers" - this makes me things of wrapper functions from Python, but maybe I have a wrong understanding of what wrapper is.

Reply:
Nice! I will move it to the engine to not confuse anyone.